### DISCUSSION TITLE
Ability to overwrite language.ini in pages?

### DISCUSSION THREAD

**Discussion by richi** [datenstrom/community#140] [Ask a question] [answered]
Inserting

``` php
if ($this->yellow->page->isExisting($key)) return $this->yellow->page->get($key);
```

in core.php, line 1768 (YellowText/get)

gives us the ability to overwrite text from language.ini (like "wikiFilter" etc.) in the meta data section of page.txt

Doing the same in (YellowConfig/get) is tempting, but probably risky.

What do you think?

Happy new year everybody, richi

**Comment by markseuffert** [comment]
Hi Richi. Can you explain a little bit what this is used for, what's the problem you want to solve?

Happy new year!

**Comment by richi** [comment]
Imagine one has more than one wiki or blog under the same domain. Right now all automatically generated pages will have the same title text etc.. For example "Wiki: Recent Changes" or whatever translation you have in your language.ini. With this little modification one can include

```
    wikiFilter: Articles
    recentChanges: What we did Yesterday 
```

in the pages.txt for one of the wikis and the text for the "Wiki: Recent Changes" page in this particular wiki will be "Articles: What we did Yesterday" without harming the other wikis.

Cheers, richi

**Comment by markseuffert** [comment, marked as answer]
You could also write a plugin that overwrites text in function `onParseMeta()`, this should give you the same functionality without changing the core. You could also change [templates and snippets](https://github.com/datenstrom/yellow/wiki/Yellow-customisation#templates) that the wiki or blog uses. That's usually the first place where people make changes... you can add your own HTML code or text strings there. 

What do you think?

**Comment by richi** [comment]
You are absolutely right. I just wanted to suggest this little change for one of the next versions of the core plugin.

Cheers, richi

**Comment by markseuffert** [comment]
Good idea. Let's wait and see if more people need this feature, then we can add it to the core.

Feedback like this is much appreciated, thank you very much! Do you have more suggestions or something not working well? Please let us know.

**Comment by richi** [comment]
Here we go: This small plugin delivers the functionality without changing the core.

https://github.com/richi/yellowcms-pagetxt

Cheers, richi

**Comment by markseuffert** [comment]
Nice work. :+1: 

Do you have plans for more plugins?
